### PR TITLE
Abbrfullname

### PR DESCRIPTION
**Pull request by 0xmtn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,95 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+// "J. R. R. Tolkien"
+
+fullname_word_list.abbrFullName(false, true); // abbreviate surname too, but not name
+// "John R. R. T."
+
+fullname_word_list.abbrFullName(true, true); // abbreviate name and surname too
+// "J. R. R. T."
+```
+
+## #07 - "use strict" and get lazy
```
Comment (loverajoel) [inline README.md:50]: you have an error here

Comment (0xmtn) [inline README.md:50]: Any error other than args object not being used?  - I just fixed it.

**Comment by 0xmtn** [comment]
ping

File: README.md [hunk 2]
```diff
@@ -18,6 +18,121 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+//John R. R. T.
+abbrFullName(fullname_word_list, {name: true, surname: true}); //abbreviate name and surname too
+//J. R. R. T.
+```
+
+By using Array.prototype: 
+###### STRONGLY DISCOURAGED!!
+
+```javascript
+Array.prototype.abbrFullName = function(args){
```
Comment (loverajoel) [inline README.md]: extend native objects it's a bad practice

**Comment by zenopopovici** [comment]
@0xmtn we can't accept this right now. Please submit a new PR. In the coming days, rules will be updated and PRs will be voted by the community before becoming tips. See #251

**Pull request closed without merging**